configure.ac

```diff
@@ -171,6 +171,22 @@ AC_ARG_WITH([backend],
   [want_backend=auto]
 )
 
+case $host in
+  *darwin*)
+    AC_PATH_PROG([BREW],brew,)
+    if test x$BREW != x; then
+      # These Homebrew packages may be keg-only, meaning that they won't be found
+      # in expected paths because they may conflict with system files. Ask
+      # Homebrew where each one is located, then adjust paths accordingly.
+      gmp_prefix=`$BREW --prefix gmp 2>/dev/null`
+      if test x$gmp_prefix != x; then
```

Comment:
better to have "" around variables, such as `if test "x$gmp_prefix" != x; then` but that's moved code, so let it be as it is